src/index.js

```diff
@@ -155,6 +157,12 @@ export default class extends PureComponent {
       // Signal this.loop function that values changed
       this.valuesChanged = true;
     }
+
+    // Refresh the Background Canvas if refreshBackgroundImage is true
+    if (this.props.refreshBackgroundImage) {
+        this.drawImage();
+    }
```

Comment:
I'd argue that we could loose the `refreshBackgroundImage` prop and actually always update the background image, if it changed. So it would look like this here:

```suggestion
    if (prevProps.imgSrc !== this.props.imgSrc) {
      this.drawImage();
    }
```